src/core/device-interface.h

```diff
@@ -7,6 +7,8 @@
 #include "tagged-profile.h"
 #include "stream-profile.h"
 
+#include <src/features-container.h>
```

Comment:
All the rest of the containers are in `core/`

Reply:
Moved